README.md

```diff
@@ -1 +1,170 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+React library
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML/CSS/JavaScript</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>GitHub</summary>
+  <ul>
+    <li><a href="https://github.com/">Github</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Real time Data about nations of the world**
+- **Most up to date countries information**
+- **clean UI design**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- a browser
+- git
+- nodejs
+
+### Setup
+
+Clone this repository to your desired folder:
+
+````sh
+  cd my-folder
+  git clone https://github.com/supreeti/Metric-webapp.git
+
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd world-Metrics-service
+  npm install
+  npm run build
+  npm start
+````
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  Open build/index.html in browser
+  or
+  Open with Live server in Vscode
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm run test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Supreeti**
+
+- GitHub: [@supreeti](https://github.com/supreeti)
+- Twitter: [@Ksupreeti](https://twitter.com/Ksupreeti)
+- LinkedIn: [Supreeti](https://www.linkedin.com/in/supreeti-kushwaha-23336232/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Users Login form and page**
+- **fully interactive SPA**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [https://github.com/supreeti/Metric-webapp.git](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me a follow and/or a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks to Original design idea by [https://www.behance.net/sakwadesignstudio](Nelson Sakwa on Behance.)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/supreeti/Metric-webapp/blob/main/LICENSE) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Please ensure you record a (not more than) 5 min video presenting the features of this project, and add the presentation link in your README file 👍🏽

Reply:
Please ensure your app is deployed and available online, and add the deployed link to your README file 😉